### PR TITLE
LPS-199445 fix test in DDMDataProviderInstancesDataProviderTest | master

### PR DESCRIPTION
**Pull request by leilanyaragao**
Jira Issue: https://liferay.atlassian.net/browse/LPS-199445

### CODE REVIEW

**Comment by leilanyaragao** [comment]
ci:test:relevant

**Comment by leilanyaragao** [comment]
ci:test:object

**Comment by marcelabc** [comment]
Hey @leilanyaragao,
Since this code is from DDM, can you reopen this PR in https://github.com/liferay-forms/liferay-portal/pulls please?
Thank you!

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-instance/src/test/java/com/liferay/dynamic/data/mapping/data/provider/instance/internal/DDMDataProviderInstancesDataProviderTest.java [hunk 1]
```diff
@@ -117,10 +116,12 @@ public void testThrowException() throws Exception {
...
 		).build();
 
+		long[] groups = {};
+
 		Mockito.when(
 			_portal.getCurrentAndAncestorSiteGroupIds(1)
-		).thenThrow(
-			PortalException.class
+		).thenReturn(
+			groups
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-instance/src/test/java/com/liferay/dynamic/data/mapping/data/provider/instance/internal/DDMDataProviderInstancesDataProviderTest.java:124]: Is it impossible to inline this?

**Pull request closed without merging**